tests/unit/test_config.py

```diff
@@ -983,19 +975,29 @@ def test_apply_vm_profiles_config_success(self):
         '''
         Tests passing in valid provider and profile config files successfully
         '''
-        providers = {'test-provider':
-                         {'digitalocean':
-                              {'driver': 'digitalocean', 'profiles': {}}}}
-        overrides = {'test-profile':
-                         {'provider': 'test-provider',
-                          'image': 'Ubuntu 12.10 x64',
-                          'size': '512MB'},
-                     'conf_file': PATH}
-        ret = {'test-profile':
-                   {'profile': 'test-profile',
-                    'provider': 'test-provider:digitalocean',
-                    'image': 'Ubuntu 12.10 x64',
-                    'size': '512MB'}}
+        providers = {
+            'test-provider': {
+                'digitalocean': {
+                    'driver': 'digitalocean', 'profiles': {}
+                }
+            }
+        }
+        overrides = {
+            'test-profile': {
+                'provider': 'test-provider',
+                'image': 'Ubuntu 12.10 x64',
+                'size': '512MB'
+            },
+            'conf_file': PATH
+        }
+        ret = {
+            'test-profile': {
+                'profile': 'test-profile',
+                'provider': 'test-provider:digitalocean',
+                'image': 'Ubuntu 12.10 x64',
+                'size': '512MB'
+            }
+        }
```

Comment:
Same here - I'd probably leave it, but if not, I'd definitely add trailing commas.